src/index.js

```diff
@@ -97,6 +97,14 @@ export default class Sound extends React.Component {
           sound.setPosition(this.props.position);
         }
       }
+
+      if (this.props.playbackRate !== prevProps.playbackRate) {
```

Comment:
You should add this to the propTypes. Also, mind adding some way of changing this in the example?